src/lib/atoms.old.ts

```
@@ -0,0 +1,1954 @@
+    const logger = createLogger('StravaActions')
+
+    switch (action.type) {
+      case 'SYNC_ACTIVITY':
+        if (!action.payload) {
+          logger.error('SYNC_ACTIVITY action requires payload with activityId')
+          return
+        }
+        const { activityId, syncAsWorkout = true } = action.payload
+
+        // Set syncing state
+        set(syncProgressAtom, prev => ({
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)